src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,185 @@
+<!---{
+  "preview": "default",
+  "default": "preview"
+}--->
+<!-- ## Introduction -->
+<!--
+This is a sample showing how to implement a seatmap that can be used for theaters, cinemas, airplanes and more.
+We are showing how to pan & zoom on the seatmap to better select seats, how to keep the availability up to date and
+how to dinamically select and unselect seats.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>SeatMap</title>
+    <link rel="canonical" href="<%host%>/advanced/seatmap.html" >
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <!-- ## Setup -->
+    <!-- Additionally used AMP components must be imported in the header: `amp-pan-zoom` for making the seatmap zoomable and scrollable -->
+    <script async custom-element="amp-pan-zoom" src="https://cdn.ampproject.org/v0/amp-pan-zoom-0.1.js"></script>
+    <!-- `amp-bind` for enabling seat selection -->
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <!-- `amp-list` for showing a list of seats -->
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+    <!-- `amp-mustache` for rendering the `amp-list` content -->
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <style amp-custom>
+      :root {
+      --abe-light-pink: rgba(182, 8, 69, 0.5);
+      --abe-pink: rgb(182, 8, 69);
+      }
+      .seat {
+        stroke-width: 3;
+        stroke: rgb(0,0,0);
+        fill:white;
+      }
+      .unavailable {
+        fill: var(--abe-light-pink);
+        pointer-events: none;
+      }
+      .seat.selected {
+        fill: var(--abe-pink);
+      }
+      .disabled-seat {
+        pointer-events: none;
+        cursor: default;
+      }
+      .content {
+        background: white;
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column;
+        border-radius: 5px;
+      }
+      .disabled {
+        pointer-events: none;
+        opacity: 0.5;
+        cursor: default;
+      }
+      .hidden {
+        display: none;
+      }
+      .zoom-button {
```

Comment:
the button? yes we do because the change that adds button by default is not yet on prod, I will update the sample once it will be, but for now we need it